models/template.go

```diff
@@ -28,6 +28,18 @@ type Question struct {
 	Order   int                `bson:"order"`
 }
 
+type QuestionView struct {
+	ID      primitive.ObjectID `bson:"_id,omitempty"`
+	Label   string             `bson:"label"`
+	Type    QuestionType       `bson:"type"`
+	Options []string           `bson:"options,omitempty"`
+	Min     int                `bson:"min,omitempty"`
+	Max     int                `bson:"max,omitempty"`
+	Order   int                `bson:"order"`
+	Error   string             `bson:"error,omitempty"`
+	Value   string             `bson:"value,omitempty"`
+}
```

Comment:
```suggestion
type QuestionView struct {
  Question
}
```

Might save a bunch of boilerplate